### DISCUSSION TITLE
Game studdenly "sped up" and lost audio quality

### DISCUSSION THREAD

**Discussion by Quills245** [ppy/osu#25072] [Q&A] [answered]
### Type

Performance

### Bug description

A while ago i just opened the game and the audio was sounding really bad and everything looks considerably faster ~~while really laggy~~ (actually the fps is fine, forgot that the lag was just cuz of the recording, the problem is just the weird sound and speed), it just began out of nowhere as far as i know, it was working normally before and right after updating to 2023.1008.1

I tried reopening the game, rebooting my pc and changing render from openGL legacy to normal openGL, none of these changed anything

And i use linux

### Screenshots or videos

https://github.com/ppy/osu/assets/99212613/f712d801-ec4c-4d68-a033-7f234143bd76

(the recording degraded the performance a bit but i think the abnormalities are still visible)

### Version

2023.1008.1

### Logs

[database.log](https://github.com/ppy/osu/files/12851506/database.log)
[input.log](https://github.com/ppy/osu/files/12851507/input.log)
[network.log](https://github.com/ppy/osu/files/12851508/network.log)
[performance.log](https://github.com/ppy/osu/files/12851509/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12851510/runtime.log)

**Comment by Quills245** [comment]
Right when i finally got some time to play this happens...

**Reply by peppy** [reply]
You can consider switching to windows which will 100% work.

As it stands, you'll have to figure out where your linux system misconfiguration is.

**Reply by Quills245** [reply]
Hmm so it's a system miscofig huh, well I'll try to look into that later then, switching to windows is something I'm not doing tho. Anyway, thank you!

**Comment by Quills245** [comment, marked as answer]
> You can consider switching to windows which will 100% work.
> 
> As it stands, you'll have to figure out where your linux system misconfiguration is.

Switching to the unnoficial flatpak release seems to have fixed it!

I guess That's more of a workaround than a fix, but it's enough for me

**Reply by Quills245** [reply]
now which comment should i mark as answer?
